TurkishFilter/sections/specific.txt

```diff
@@ -282,7 +282,7 @@ havadiskibris.com##img[width="710"][height="80"]
 ||haberkibris.com/banner
 haberkibris.com##img[alt="ads"]
 mynet.com##div[issponsored]
-||anizm.net/js/po-under.js
+/po-under.js$domain=anizle.com|anizm.net
```

Comment:
Also please add a new domain to https://github.com/AdguardTeam/AdguardFilters/blob/c8a023cfa29a0a4ec1bb23e5c3440612cc01b89e/TurkishFilter/sections/general_extensions.txt#L15